source/hint_list.cpp

```diff
@@ -2067,6 +2081,51 @@ void HintTable_Init() {
+                       Text{"the #third or fourth game chest# contains", /*french*/"le #troisième ou quatrième coffre à jeu# contient", /*spanish*/"#los juegos de azar# revelan [2]"},
+                     }, {},
+                       //clear text
+                       Text{"the #second locked room# in the chest game contains", /*french*/"la #deuxième salle# de la chasse aux trésors contient", /*spanish*/"#el Cofre del Tesoro# premia [2]"}
+  );
+
+    hintTable[MARKET_TREASURE_CHEST_GAME_ITEM_3] = HintText::Sometimes({
+                       //obscure text
+                       Text{"#gambling 3 times# grants",                /*french*/"#parier trois fois# donne",                       /*spanish*/"#los juegos de azar# revelan [3]"},
+                       Text{"the #fifth or sixth game chest# contains", /*french*/"le #cinquième ou sixième coffre à jeu# contient", /*spanish*/"#los juegos de azar# revelan [3]"},
+                     }, {},
+                       //clear text
+                       Text{"the #third locked room# in the chest game contains", /*french*/"la #troisième salle# de la chasse aux trésors contient", /*spanish*/"#el Cofre del Tesoro# premia [3]"}
+  );
+
+    hintTable[MARKET_TREASURE_CHEST_GAME_ITEM_4] = HintText::Sometimes({
+                       //obscure text
+                       Text{"#gambling 4 times# grants",                   /*french*/"#parier quatre fois# donne",                      /*spanish*/"#los juegos de azar# revelan [4]"},
+                       Text{"the #seventh or eighth game chest# contains", /*french*/"le #septième ou huitième coffre à jeu# contient", /*spanish*/"#los juegos de azar# revelan [4]"},
+                     }, {},
+                       //clear text
+                       Text{"the #fourth locked room# in the chest game contains", /*french*/"la #quatrième salle# de la chasse aux trésors contient", /*spanish*/"#el Cofre del Tesoro# premia [4]"}
+  );
+
+    hintTable[MARKET_TREASURE_CHEST_GAME_ITEM_5] = HintText::Sometimes({
+                       //obscure text
+                       Text{"#gambling 5 times# grants",                /*french*/"#parier cinq fois# donne",                       /*spanish*/"#los juegos de azar# revelan [5]"},
+                       Text{"the #ninth or tenth game chest# contains", /*french*/"le #neuvième ou dixième coffre à jeu# contient", /*spanish*/"#los juegos de azar# revelan [5]"},
+                     }, {},
+                       //clear text
+                       Text{"the #fifth locked room# in the chest game contains", /*french*/"la #cinquième salle# de la chasse aux trésors contient", /*spanish*/"#el Cofre del Tesoro# premia"}
+  );
+
     hintTable[GF_HBA_1500_POINTS] = HintText::Sometimes({
                        //obscure text
                        Text{"mastery of #horseback archery# grants", /*french*/"maîtriser l'#archerie équestre# donne", /*spanish*/"amaestrar el #tiro con arco a caballo# premia"},
```

Comment:
Text{"mastery of #horseback archery# grants", /*french*/"maîtriser l'#archerie équestre# donne", /*spanish*/"dominar el #tiro con arco a caballo# premia con"},